org-roam.el

```diff
@@ -1152,45 +1152,54 @@ Applies `org-roam-link-face' if PATH corresponds to a Roam file."
 
 ;;;; org-roam-backlinks-mode
 (defvar org-roam-backlinks-mode-map
-  (let ((km (make-sparse-keymap)))
-    (define-key km [mouse-1] 'org-roam-open-at-point)
-    (define-key km (kbd "RET") 'org-roam-open-at-point)
-    km)
+  (let ((map (make-sparse-keymap)))
+    (define-key map [mouse-1] 'org-open-at-point)
+    (define-key map (kbd "RET") 'org-open-at-point)
```

Comment:
I'm still keeping the bindings, as by default in Org mode `RET` is bound to `org-return`, which would work correctly for the "visit links" feature, but would not work for the "visit preview text" feature, as `org-return` only calls `org-open-at-point` if both  `org-return-follows-link` is set _and_ point is on a link.